projects/tinkerbell/hook/patches/0004-support-overriding-linuxkit-version-to-use-eksa-buil.patch

```diff
@@ -0,0 +1,25 @@
+From 047cafd4913207911ca25a252eec599da13c4448 Mon Sep 17 00:00:00 2001
+From: Prow Bot <prow@amazonaws.com>
+Date: Mon, 7 Oct 2024 19:38:02 +0000
+Subject: [PATCH 4/4] support overriding linuxkit version to use eksa build
+
+---
+ build.sh | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/build.sh b/build.sh
+index bebfa11..b45d48c 100755
+--- a/build.sh
++++ b/build.sh
+@@ -29,7 +29,7 @@ declare -g HOOK_LK_CONTAINERS_OCI_BASE="${HOOK_LK_CONTAINERS_OCI_BASE:-"quay.io/
+ declare -g SKOPEO_IMAGE="${SKOPEO_IMAGE:-"quay.io/skopeo/stable:latest"}"
+ 
+ # See https://github.com/linuxkit/linuxkit/releases
+-declare -g -r LINUXKIT_VERSION_DEFAULT="1.5.0" # LinuxKit version to use by default; each flavor can set its own too
++declare -g -r LINUXKIT_VERSION_DEFAULT="${LINUXKIT_VERSION_DEFAULT:-1.5.0}" # LinuxKit version to use by default; each flavor can set its own too
```

Comment:
I meant to do this change earlier, this just makes sure that when we run linuxkit we are using the one built from eksa.